docs/products/clickhouse/howto/enable-tiered-storage.rst

```diff
@@ -0,0 +1,83 @@
+Enable tiered storage in Aiven for ClickHouse® |beta|
+=====================================================
+
+Learn how to enable the tiered storage feature on your Aiven for CliockHouse® service and activate it for specific tables.
+
+.. seealso::
+
+    Check what the tiered storage is, how it works, and why use it in :doc:`Tiered storage in Aiven for ClickHouse® </docs/products/clickhouse/concepts/clickhouse-tiered-storage>`.
+
+About enabling tiered storage
+-----------------------------
+
+You can enable the tiered storage feature in Aiven for ClickHouse either for a new service or for an existing one.
+To enable the feature, you need to activate it on a service level in Aiven Console and, next, set it up on a table level, which is done using CLI (SQL).
```

Comment:
The SQL part is more important, and seeing it can be run using different clients (web console, dedicated client, cli) it would make more sense to highlight SQL and give CLI as an example?